app/templates/macros/toolkit_forms.html

```diff
@@ -65,32 +74,32 @@
   ) }}
 {%- endmacro %}
 
-{% macro upload(question_content, service_data, errors) -%}
-  {{ upstream_forms.upload(question_content, service_data, errors,
+{% macro pricing(question_content, service_data, errors) -%}
+  {{ upstream_forms.pricing(question_content, service_data, errors,
```

Comment:
Is this reordering to match the order in another file? If there isn't a clear good reason for it I'd argue it's best not to muddy the blame history of the file.

Reply:
I reordered it so the one macro that had the unique kwargs (upload having an additional value kwarg) would be somewhat separate.

After your comment and in retrospect I'm going to undo that change.